src/commands/containers/inspectContainer.ts

```diff
@@ -9,7 +9,10 @@ import { ContainerTreeItem } from "../../tree/containers/ContainerTreeItem";
 
 export async function inspectContainer(context: IActionContext, node?: ContainerTreeItem): Promise<void> {
     if (!node) {
-        node = await ext.containersTree.showTreeItemPicker<ContainerTreeItem>(ContainerTreeItem.allContextRegExp, context);
+        node = await ext.containersTree.showTreeItemPicker<ContainerTreeItem>(ContainerTreeItem.allContextRegExp, {
+            ...context,
+            noItemFoundErrorMessage: 'No containers are available to inspect'
```

Comment:
Nit: some error messages end with a period, some don't. Let's be consistent. Suggest no period

Reply:
fixed